cpp/src/gandiva/precompiled/time_test.cc

```diff
@@ -839,4 +839,86 @@ TEST(TestTime, TestToTimeNumeric) {
   EXPECT_EQ(expected_output, to_time_float64(3601.500));
 }
 
-}  // namespace gandiva
+TEST(TestTime, TestFromUnixtimeWithoutPattern) {
```

Comment:
Create some integrated tests for this function in Java or in the `tests` folder

Reply:
Ok, will do it.